src/histolab/masks.py

```diff
@@ -152,3 +155,23 @@ def _mask(self, slide) -> np.ndarray:
         filters = FiltersComposition(histolab.slide.Slide).tissue_mask_filters
         thumb_mask = filters(thumb)
         return thumb_mask
+
+    @lru_cache(maxsize=100)
+    @_mask.register(Tile)
+    def _(self, tile: Tile) -> np.ndarray:
+        """Return the thumbnail binary mask of the tissue area.
```

Comment:
fair enough

Reply:
I tried to redefine the `__call__` method in the `TissueMask` class, but I'm struggling to put type hints:

- if I do `from .slide import Slide` it fails due to circular imports
- with both`import histolab` and `import histolab.slide` I get `AttributeError: module 'histolab' has no attribute 'slide'`
- with `from histolab import slide` I would get a LGTM alert because the module is already imported as `import histolab`

I could just leave it without annotation and explain it only in the docstring, but I don't like it, any help @ernestoarbitrio ?

Reply:
I think define `_` is ok, and i don't think it will be rendered in the docs ... anyhow the docstring should contain something meaningful for future development